usl_pipeline/cloud_functions/main_test.py

```diff
@@ -165,12 +165,16 @@ def test_build_feature_matrix_wrf(
     time = ncfile.createVariable("time", "f8", ("Time",))
     lat = ncfile.createVariable("lat", "float32", ("west_east",))
     lon = ncfile.createVariable("lon", "float32", ("south_north",))
-    # Create dataset entries for all variables in mock
+    # Create dataset entries for all variables in mock. Order of the lat/lon dims
```

Comment:
Why does swapping the lat lons make for easier testing? Is that why we added the `feature = feature.transpose("west_east", "south_north", ...)` call above? I'd generally recommend testing with data that looks as much like the real data as possible.

Reply:
Ack - I agree on using real-ish data during test. My rationale was that it would help separate testing concerns since `process_wps_feature` will perform a series of transforms that are additive (for ex: reordering axis), I wanted to make it easier to isloation the transform logic that we were testing.

I can refacotr to make all the test datasets mimic the real inputs though